File: C11/백준/Bronze/1075. 나누기/나누기.c
#include <stdio.h>

int main(void) {
	
	int N, F;
	scanf("%d %d", &N, &F);
	
	int temp = (N / 100) * 100;
	int result = (temp % F == 0) ? temp : temp + (F - (temp % F));
	
	printf("%02d", result % 100);
	
	return 0;
}

File: Java/백준/Bronze/1075. 나누기/나누기.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int N = Integer.parseInt(br.readLine());
        int F = Integer.parseInt(br.readLine());

        int temp = (N / 100) * 100;
        int result = (temp % F == 0) ? temp : temp + (F - (temp % F));

        System.out.printf("%02d", result % 100);
    }
}